client/src/lib/src/types/requests.ts

```diff
@@ -8,17 +8,20 @@ import { type EmbeddedLanguageType } from './embedded-languages'
 
 export enum RequestType {
   EmbeddedLanguageTypeOnPosition = 'EmbeddedLanguageTypeOnPosition',
-  getLinksInDocument = 'getLinksInDocument'
+  getLinksInDocument = 'getLinksInDocument',
+  ProcessRecipeScanResults = 'ProcessRecipeScanResults'
 }
 
 export const RequestMethod: Record<RequestType, string> = {
-  [RequestType.EmbeddedLanguageTypeOnPosition]: 'custom/requestEmbeddedLanguageTypeOnPosition',
-  [RequestType.getLinksInDocument]: 'custom/getLinksInDocument'
+  [RequestType.EmbeddedLanguageTypeOnPosition]: 'custom/requestEmbeddedLanguageDocInfos',
+  [RequestType.getLinksInDocument]: 'custom/getLinksInDocument',
+  [RequestType.ProcessRecipeScanResults]: 'custom/ProcessRecipeScanResults'
 }
 
 export interface RequestParams {
   [RequestType.EmbeddedLanguageTypeOnPosition]: { uriString: string, position: Position }
   [RequestType.getLinksInDocument]: { documentUri: string }
+  [RequestType.ProcessRecipeScanResults]: { scanResults: string }
```

Comment:
IMO should be in the client:
# Less data exchanges for better performance
# Mirrors the notification type that projectScanner uses (notification instead of request). More likeness = easier to maintain
# Mirrors the implementation of projectScanner

Reply:
As you mentioned in another conversation, I will implement the parse function in the bitbakeRecipeScanner class.

Reply:
Closing this conversation as I am implementing the parsing in the `analzyer`